src/main/scala/org/reactivestreams/Publisher.scala

```diff
@@ -0,0 +1,29 @@
+package org.reactivestreams
```

Comment:
Technically won't be in project as the "server side" jvm interfaces, but do we want to consider a sub-package in any case?